src/docs/output-targets/custom-elements.md

```diff
@@ -33,6 +34,21 @@ customElements.define('hello-world', HelloWorld);
 
 The generated files will export each component class and will already have the styles bundled. However, it does not define the custom elements or apply any polyfills. Any dependencies of your imported component will need to be loaded as well. 
 
+## Config
```

Comment:
I intentionally didn't turn this into a table, as the readability was pretty gross in the MD file itself